compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/BaseComposeScene.skiko.kt

```diff
@@ -145,12 +145,14 @@ internal abstract class BaseComposeScene(
     }
 
     override fun render(canvas: Canvas, nanoTime: Long) = postponeInvalidation {
-        recomposer.performScheduledTasks()
-        frameClock.sendFrame(nanoTime) // Recomposition
+        snapshotInvalidationTracker.rendering {
```

Comment:
It's already wrapped in `postponeInvalidation`. Can we combine it somehow?

Reply:
Agree, better to move to/combine with `postponeInvalidation`. Because we also need to perform changes not only during rendering, but also during an event handling.

Reply:
I considered that, but didn't want to change more than necessary. If you think we should perform the changes synchronously whenever we're inside `postponeInvalidation` then yes, we should combine the two.

Reply:
> we should perform the changes synchronously whenever we're inside postponeInvalidation then yes, we should combine the two.

Yes, the logic of hanlding events should be similar to rendering the frames. They both change states and cause side effects.